src/test/regress/expected/distributed_functions.out

```diff
@@ -31,6 +31,21 @@ SELECT * FROM run_command_on_workers('SELECT function_tests.add(2,3);') ORDER BY
  localhost |    57638 | t       | 5
 (2 rows)
 
+-- change the owner of the function and verify the owner has been changed on the workers
+ALTER FUNCTION add(int,int) OWNER TO functionuser;
```

Comment:
Maybe add some `ALTER PROCEDURE .. OWNER TO .` tests as well.

I manually tested and saw that they do not work yet